.github/workflows/update_bundle.yaml

```diff
@@ -3,13 +3,12 @@
 name: Update bundle
 
 on:
-  schedule:
-    - cron: '53 0 * * *'  # Daily at 00:53 UTC
+  workflow_dispatch:
 
 jobs:
   update-bundle:
     name: Update bundle
-    uses: canonical/data-platform-workflows/.github/workflows/update_bundle.yaml@v21.0.1
+    uses: canonical/data-platform-workflows/.github/workflows/update_bundle.yaml@lucas/add-rock-pinning
```

Comment:
this was added here because of a custom run I did on https://github.com/canonical/postgresql-k8s-bundle/pull/193/files